code/_onclick/hud/hud.dm

```diff
@@ -23,6 +24,15 @@
 	var/obj/screen/move_intent
 	var/obj/screen/stamina/stamina_bar
 	var/obj/screen/meter/health/hud_healthbar
+	var/obj/screen/meter_component/current/remaining_meter	//The actual remaining health, in red or green
+	var/obj/screen/meter_component/delta/delta_meter	//A yellow section indicating recent loss
+	var/obj/screen/meter_component/limit/limit_meter	//A solid grey block at the end, representing reduced maximum
+	var/obj/screen/meter_component/text/textholder
+	var/obj/screen/meter/resource/hud_resource
```

Comment:
this is really bad, we can't be having all these variables
these meters need to be in a list, and the parent object responsible for all the components

i'm aware it was already kinda bad, but this makes it worse

Reply:
Updated